admob/integration_test/src/integration_test.cc

```diff
@@ -937,6 +937,9 @@ TEST_F(FirebaseAdMobTest, TestBannerView) {
             bounding_box_listener.bounding_box_changes_.size());
 
 #if defined(ANDROID) || TARGET_OS_IPHONE
```

Comment:
Instead of disabling this test, is it possible to just add some wiggle room to the aspect ratio check on simulator?